apps/meteor/app/api/server/v1/instances.ts

```diff
@@ -17,18 +15,43 @@ API.v1.addRoute(
 			const instances = await InstanceStatus.find().toArray();
 
 			return API.v1.success({
-				instances: instances.map((instance: IInstanceStatus) => {
-					const connection = getInstanceConnection(instance);
-
-					if (connection) {
-						delete connection.instanceRecord;
-					}
-					return {
-						...instance,
-						connection,
-					};
-				}),
+				instances,
+				// instances: instances.map((instance: IInstanceStatus) => {
+				// 	const connection = getInstanceConnection(instance);
+
+				// 	if (connection) {
+				// 		delete connection.instanceRecord;
+				// 	}
+				// 	return {
+				// 		...instance,
+				// 		connection,
+				// 	};
+				// }),
 			});
 		},
 	},
 );
+
+// TODO why?
+// function getConnection(address) {
```

Comment:
I believe we used this endpoint for debug only, so it was returning all the relevant information about the instances and the connection status